nextflow_schema.json

```
@@ -87,150 +76,193 @@
         "run_ids": {
           "type": "string",
           "default": "All",
-          "description": "Specify a comma-separated list of run IDs to run. Use \"All\" to run all samples in the run_metafile."
+          "description": "Run IDs",
+          "help_text": "A comma-separated list of run IDs to run. Use \"All\" to run all samples in the run_metafile."
         },
         "project": {
           "type": "string",
-          "description": "Run samples only from a given project ID"
+          "description": "Project ID",
+          "help_text": "Run samples only from a given project ID"
```

```suggestion
          "help_text": "A comma-separated list of project IDs. All samples with the provided project IDs will be run."
```
This should also be updated to reflect that you can provide multiple IDs.